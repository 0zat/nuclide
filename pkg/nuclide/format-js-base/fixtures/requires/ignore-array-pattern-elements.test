'use strict';

var [a, b] = foo;
function bar([c, d]) {}
