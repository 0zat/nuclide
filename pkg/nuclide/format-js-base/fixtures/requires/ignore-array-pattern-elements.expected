'use strict';

var foo = require('foo');

var [a, b] = foo;
function bar([c, d]) {}
